.github/workflows/ci.yml

```diff
@@ -249,8 +255,34 @@ jobs:
       - name: Test containers are up
         run: |
           curl http://localhost:8080/bitcoin -d '{"jsonrpc": "1.0", "method": "sendtoaddress", "params": ["bcrt1qylgu6ffkp3p0m8tw8kp4tt2dmdh755f4r5dq7s", 0.1]}'
-      - name: Run coordinator, maker, etc
```

Comment:
❓ Why do `just coordinator`  and `just maker` not work?

Reply:
It would rebuild the binaries even though they are in `target/debug`

Reply:
We could split it into `build-coordinator` and `run-coordinator` and then `coordinator` would call both. Then you could use `run-coordinator` here.

Not a huge deal, but it might be worth it to avoid duplication.

Reply:
I'd do this when we have to touch it again.